core/services/relay/evm/types/types.go

```diff
@@ -40,9 +40,33 @@ type FunctionsProvider interface {
 	relaytypes.PluginProvider
 	LogPollerWrapper() LogPollerWrapper
 }
+type OracleRequest struct {
+	OracleContract     []byte
+	RequestId          []byte
+	RequestingContract []byte
+	RequestInitiator   []byte
+	SubscriptionId     uint64
+	SubscriptionOwner  []byte
+	Data               []byte
+	DataVersion        uint16
+	Flags              []byte
+	CallbackGasLimit   uint64
+	TxHash             []byte
+}
+
+type OracleResponse struct {
+	RequestId   []byte
+	Transmitter []byte
+	Type        string
```

Comment:
Added Type field for the name of the event. Used to give prometheus a label of what happened